quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/update/Quarkus213to38CliUpdateIT.java

```diff
@@ -14,7 +15,8 @@
 import io.quarkus.test.scenarios.annotations.EnabledOnQuarkusVersion;
 import io.quarkus.test.utils.AwaitilityUtils;
 
-@EnabledOnQuarkusVersion(version = "3.8.*", reason = "This class is testing only updates to 3.8.* versions")
+@Tag("quarkus-cli")
+@EnabledOnQuarkusVersion(version = "3.8.*redhat.*", reason = "This class is testing only updates to 3.8.* RHBQ versions")
```

Comment:
This addresses: 
```
2024-09-03T08:30:19.5006021Z [ERROR] io.quarkus.ts.quarkus.cli.update.Quarkus213to38CliUpdateIT.versionUpdateTest -- Time elapsed: 62.54 s <<< FAILURE!
2024-09-03T08:30:19.5008078Z org.opentest4j.AssertionFailedError: Updated app is not using "redhat" version. Found version: 3.8.6 ==> expected: <true> but was: <false>
2024-09-03T08:30:19.5010152Z 	at io.quarkus.ts.quarkus.cli.update.AbstractQuarkusCliUpdateIT.versionUpdateTest(AbstractQuarkusCliUpdateIT.java:72)
2024-09-03T08:30:19.5011643Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
2024-09-03T08:30:19.5012579Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
2024-09-03T08:30:19.5013480Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
```
So after looking at tests, I think they are creating application with community and expecting RHBQ.

I'll approve this know, but so that you know, I think we should write tests in a way they work for both community and RHBQ. I can imagine that **in the future**, you add condition based on created application, not hardcoded version. So if you have only community registries, you would expect community and conversely.

Reply:
> So after looking at tests, I think they are creating application with community and expecting RHBQ.

Quarkus CLI works in a way, that if you have RHBQ registry setup it will create RHBQ app, even if you specify something like `--stream=3.2`. The same if you update.

I can add some condition to only expect app to be *.redhat version only if running with RHBQ, that should work.

Reply:
I don't think we understand each other. What I meant is that here, test was failing because we run tests with community registry. Therefore I would expect condition to be dynamic. If I have infer from `quarkus.platform.group-id` I am testing community, I'd expect community.

Anyway, I understand you. We can re-discuss it for future development only (if I am reviewer).